packages/loader/container-loader/src/container.ts

```diff
@@ -1082,8 +1091,8 @@ export class Container
 				this._protocolHandler?.close();
 
 				this.connectionStateHandler.dispose();
-			} catch (exception) {
-				this.mc.logger.sendErrorEvent({ eventName: "ContainerCloseException" }, exception);
+			} catch (error_) {
+				this.mc.logger.sendErrorEvent({ eventName: "ContainerCloseException" }, error_);
```

Comment:
So the rule is looking specifically for `error*`? I thought it just didn't like `e` or `err`.

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v48.0.1/docs/rules/catch-error-name.md

The linter complains when changing back to `exception`. Didn't know such rule exists.

Reply:
Huh, this was an auto-fix, but I'm surprised by the `_` postfix. I'll investigate. Ideally it wants `error`, but I'm guessing that already exists in an outer scope.